experiments/yarn/src/main/scala/org/apache/gearpump/experiments/yarn/Client.scala

```diff
@@ -48,45 +48,32 @@ Features for YARNClient
 - [ ] Client needs to use YARN cluster API to find best nodes to run Workers
  */
 
-object EnvVars {
-  val APPMASTER_NAME = "gearpump.yarn.applicationmaster.name"
-  val APPMASTER_COMMAND = "gearpump.yarn.applicationmaster.command"
-  val APPMASTER_MEMORY = "gearpump.yarn.applicationmaster.memory"
-  val APPMASTER_VCORES = "gearpump.yarn.applicationmaster.vcores"
-  val APPMASTER_QUEUE = "gearpump.yarn.applicationmaster.queue"
-  val APPMASTER_MAIN = "gearpump.yarn.applicationmaster.main"
-  val CONTAINER_COUNT = "gearpump.yarn.container.count"
-  val CONTAINER_MEMORY = "gearpump.yarn.container.memory"
-  val CONTAINER_VCORES = "gearpump.yarn.container.vcores"
-  val EXCLUDE_JARS = "gearpump.yarn.client.excludejars"
-  val HDFS_PATH = "gearpump.yarn.client.hdfsPath"
-  val JARS = "gearpump.yarn.client.jars"
-}
 
 trait ClientAPI {
```

Comment:
Is it reasonble to add some documents in the public trait?

So that the document can show up the scala doc.